packages/grub2/build.yaml

```diff
@@ -27,10 +36,10 @@ steps:
     {{if eq .Values.tool_image_distribution "opensuse"}}
       {{if .Values.arch }}
         {{if eq .Values.arch "x86_64"}}
-- cp /usr/share/grub2/x86_64-efi/grub.efi /EFI/BOOT/bootx64.efi
+- cp /usr/share/efi/x86_64/* /EFI/BOOT/
```

Comment:
here we should not care about the naming as elemental-cli already do its stuff

Reply:
Please add this knowledge as comments to the code.

Reply:
Packages are tool independent, comment is here because the package change affect us directly.

In any case this still need changes to be backwards compatible so it can't go out like this.